include/libopencm3/sam/usb.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef SAM3X_USB_H
+#define SAM3X_USB_H
+
+#if defined(SAM3A)
+#elif defined(SAM3N)
+#elif defined(SAM3S)
+#       include <libopencm3/sam/udp.h>
+#elif defined(SAM3U)
+#elif defined(SAM3X)
+#else
+#       error "sam family not defined."
+#endif
```

Comment:
What do you think about adding #error directivces for the 3a, 3u and 3x that all have usb, just not supported (yet) ?  just an idea.